test/hotspot/jtreg/runtime/StackGuardPages/exeinvoke.c

```diff
@@ -68,8 +68,17 @@ static void handler(int sig, siginfo_t *si, void *unused) {
   longjmp(context, 1);
 }
 
+static char* altstack = NULL;
+
 void set_signal_handler() {
-  static char altstack[SIGSTKSZ];
+  if (altstack == NULL) {
+    // Dynamically allocated incase SIGSTKSZ is not constant
```

Comment:
"incase" --> "in case"?
Also need to update the copyright year.

Reply:
Both fixed. Thanks for the review @DamonFool !